src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -714,6 +714,10 @@
     <value>Never close automatically</value>
     <comment>An option to choose from for the "profile termination behavior" (or "close on exit") setting. When selected, the terminal never closes, even if the process exits in a controlled or uncontrolled scenario. The user would have to manually close the terminal.</comment>
   </data>
+  <data name="Profile_CloseOnExitGracefulIfLaunchedByTerminal.Content" xml:space="preserve">
+    <value>Graceful if launched by Windows Terminal</value>
+    <comment>An option to choose from for the "profile termination behavior" (or "close on exit") setting. When selected, the terminal closes if the process exits in a controlled scenario successfully and the process was launched by Windows Terminal.</comment>

```

Comment:
Any suggestions for this value/comment would be helpful

Reply:
paging @cinnamon-msft

Reply:
Graceful doesn't mean much here (that's why the "graceful" option above doesn't use that term)

How about "Close only when process exits successfully (exclude default terminal sessions)"?
And we rename the other one to "Close only when process exits successfully (all sessions)"?

Reply:
Those strings are very long and would cause issues when the window is narrow

Reply:
consensus: "Automatic"

Reply:
Renamed to automatic